### PR TITLE
fix(challenge): account for browser zoom, add note to challenge (for height challenge)

### LINKED ISSUE #18045: Applied Visual Design: Adjust the Height of an Element Using the height Property - correct solution does not pass
#### Describe your problem and - if possible - how to reproduce it
When you provide the correct code the test still fails, this is do to the tests passing the true height of the element and not what the CSS is set to. This issue occurs mainly on smaller resolutions because the height of the element is smaller and converted by the browser.

In my Dev Tools it was saying the true height of the element was 24.99px and when I would scroll out the browser converted the true height to 25.01px. I had to find a sweet spot in my zoom level to where it was exactly 25px.

The fix is to change the test to use regex and not use `assert($('h4').css('height') == '25px'`

Additional users having issues: https://forum.freecodecamp.org/t/applied-visual-design-adjust-the-height-of-an-element-using-the-height-property/199328/5

#### Add a Link to the page with the problem
https://learn.freecodecamp.org/responsive-web-design/applied-visual-design/adjust-the-height-of-an-element-using-the-height-property/

#### Tell us about your browser and operating system

* Browser Name:  Chrome
* Browser Version: 
* Operating System:

### PR DESCRIPTION
**Pull request by lasjorg**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Second go at fixing this challenge. Forum [confirmation](https://www.freecodecamp.org/forum/t/is-there-a-glitch-in-this-challenge/325152/62) that this should work. I also added a note about the zoom.

BTW, how do we update the hint forum post?

Closes #18045

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by lasjorg** [comment]
Let's just the regex, it's safer I think.

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/adjust-the-height-of-an-element-using-the-height-property.english.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ img {
 ## Instructions
 <section id='instructions'>
 Add a <code>height</code> property to the <code>h4</code> tag and set it to 25px.
+<strong>Note:</strong> You may need to be at 100% zoom to pass the test on this challenge.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/adjust-the-height-of-an-element-using-the-height-property.english.md]:
```suggestion
```
No longer needed with the current test.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
I think I'm finally caught up on the discussions around this.  I agree that, sadly, regex is the way to go.

With that in mind:

**Review by ojeytonwilliams — APPROVED** [review]
Thanks @lasjorg, LGTM now!

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Similar problems as #37688. I can pass the challenge with the following which do not actually affect the height.
```css
  h4 {
    text-align: center;
    Height: 25px;
  }
```
OR

```css
  h4 {
    text-align: center;
    max-height: 25px;
  }
```
OR
```css
  h4 {
    text-align: center;
    myheight: 25px;
  }
```

**Comment by lasjorg** [comment]
Reverted, same argument applies as in https://github.com/freeCodeCamp/freeCodeCamp/pull/37688#issuecomment-554590238

**Review by moT01 — APPROVED** [review]
I'm fine with it like this, I tested it pretty thoroughly.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**